contracts/FraudProof.sol

```diff
@@ -35,69 +34,13 @@ contract FraudProofHelpers is FraudProofSetup {
         }
 
         // check from leaf has enough balance
-        if (_from_account.balance < amount.add(fee)) {
+        if (fromState.balance < amount.add(fee)) {
             // invalid state transition
-            // needs to be slashed because the account doesnt have enough balance
+            // needs to be slashed because the state doesnt have enough balance
             // for the transfer
             return Types.ErrorCode.NotEnoughTokenBalance;
         }
 
         return Types.ErrorCode.NoError;
     }
-
-    function RemoveTokensFromAccount(
-        Types.UserAccount memory account,
-        uint256 numOfTokens
-    ) public pure returns (Types.UserAccount memory updatedAccount) {
-        return (
-            UpdateBalanceInAccount(
-                account,
-                BalanceFromAccount(account).sub(numOfTokens)
-            )
-        );
-    }
-
-    // returns a new User Account with updated balance
-    function UpdateBalanceInAccount(
-        Types.UserAccount memory original_account,
-        uint256 new_balance
-    ) public pure returns (Types.UserAccount memory updated_account) {
-        original_account.balance = new_balance;
-        return original_account;
-    }
-
-    function AddTokensToAccount(
-        Types.UserAccount memory account,
-        uint256 numOfTokens
-    ) public pure returns (Types.UserAccount memory updatedAccount) {
-        return (
-            UpdateBalanceInAccount(
-                account,
-                BalanceFromAccount(account).add(numOfTokens)
-            )
-        );
-    }
-
-    function BalanceFromAccount(Types.UserAccount memory account)
-        public
-        pure
-        returns (uint256)
-    {
-        return account.balance;
-    }
-
-    /**
-     * @notice Returns the updated root and balance
-     */
-    function UpdateAccountWithSiblings(
```

Comment:
We no longer use this either, since this function depends on `_merkle_proof.pathToAccount`.